mysql/README.md

```diff
@@ -1,59 +1,143 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog Agent can collect many metrics from MySQL databases, including those for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And many more. You can also invent your own metrics using custom SQL queries.
```

Comment:
rather than invent your own I might suggest something like:

SQL queries can be provided to generate custom metrics.